hack/multi-node/Vagrantfile

```diff
@@ -181,6 +185,12 @@ Vagrant.configure("2") do |config|
 
       worker.vm.provision :file, :source => CA_CERT_PATH, :destination => "/tmp/ca.crt"
       worker.vm.provision :shell, :inline => "mv /tmp/ca.crt /etc/kubernetes/ca.crt", :privileged => true
+
+      worker.vm.provision :file, :source => CA_CERT_PATH, :destination => "/tmp/ca.crt"
```

Comment:
Why is this part of the PR?

Reply:
good catch... there was a conflict on a rebase... I will remove this change.